runtime/parachains/src/util.rs

```diff
@@ -39,3 +40,20 @@ pub fn make_persisted_validation_data<T: paras::Config + hrmp::Config>(
 		max_pov_size: config.max_pov_size,
 	})
 }
+
+/// Take the active subset of a set containing all validators.
+pub fn take_active_subset<T: Clone>(active_validators: &[ValidatorIndex], set: &[T]) -> Vec<T> {
+	let subset: Vec<_> = active_validators.iter()
+		.filter_map(|i| set.get(i.0 as usize))
+		.cloned()
+		.collect();
+
+	if subset.len() != active_validators.len() {
+		log::warn!(
+			target: "parachains_runtime",
```

Comment:
we do this to allow matching by prefix, e.g. https://github.com/paritytech/polkadot/blob/f02d5e45ada77621954cb7ad3893cb81de657d15/runtime/parachains/src/runtime_api_impl/v1.rs#L88

```suggestion
			target: "runtime::parachains",
```

Reply:
only on some builds https://github.com/paritytech/polkadot/pull/2522

Reply:
Uh, but default is on - that's good! So for Rococo, local tests and maybe even Westend we will have it switched on by default. I like.